test/resources/microshift-process.resource

```diff
@@ -30,9 +30,15 @@ MicroShift Is Ready
 
 MicroShift Is Live
     [Documentation]    Check the /livez endpoint
-    ${stdout}=    Run With Kubeconfig    oc get --raw='/livez'
+    [Arguments]    ${extra_args}=${EMPTY}
+    ${stdout}=    Run With Kubeconfig    oc get --raw='/livez' ${extra_args}
     Should Be Equal As Strings    ${stdout}    ok    strip_spaces=True
 
+MicroShift Is Live With Custom CA
```

Comment:
Should we better move this function to the test itself, rather than keeping it in the generic file?